infra/_modules/ai_search/providers.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
+  required_providers {
+    restapi = {
+      source  = "Mastercard/restapi"
```

Comment:
I think we cannot use these providers from third party vendors. @Krusty93 what about this?